src/components/PageHero.tsx

```diff
@@ -89,7 +93,20 @@ const StyledButtonLink = styled(ButtonLink)`
   }
 `
 
-const PageHero = ({ content, children, className, isReverse }) => {
+// TODO: fix content type, once an interface is created for the data
+export interface IProps {
+  content: any
+  isReverse?: boolean
+  children?: React.ReactNode
+  className?: string
+}
+
+const PageHero: React.FC<IProps> = ({
+  content,
+  isReverse = false,
+  children,
+  className,
+}) => {
   const { buttons, title, header, subtitle, image, alt } = content
```

Comment:
For now, we could add the content interface in this file with this structure. Could be something like:
```ts
interface IButton {
  path: string
  pathId: string
}

interface IContent {
  buttons?: Array<IButton>
  title: string
  header: string
  subtitle: string
  image: string
  alt: string
}
```